packages/schematics/src/collection/library/index.ts

```diff
@@ -369,6 +369,17 @@ function updateTsConfig(options: NormalizedSchema): Rule {
   ]);
 }
 
+function updateLibPackageNpmScope(options: NormalizedSchema): Rule {
+  return chain([
```

Comment:
It would be easier to return this instead of a chain since it already returns a rule:
```ts
return updateJsonInTree('package.json', json => {
        json.name = `@${options.prefix}/${options.name}`;
        return json;
      });
```